tests/helper/reporter/httpMeasurementReporter.go

```diff
@@ -36,6 +37,7 @@ func (r *HTTPMeasurementReporter) SpecDidComplete(specSummary *types.SpecSummary
 			output := map[string]string{}
 			output["Number of Samples"] = strconv.Itoa(specSummary.NumberOfSamples)
 			output["Measurement"] = k
+			output["PR"] = os.Getenv("PULL_NUMBER")
```

Comment:
a comment here regarding this env variable

Reply:
updated